include/mbedtls/mbedtls_config.h

```diff
@@ -2862,10 +2862,6 @@
 /**
  * \def MBEDTLS_PKCS7_C
  *
- * This feature is a work in progress and not ready for production. Testing and
- * validation is incomplete, and handling of malformed inputs may not be robust.
- * The API may change.
- *
  * Enable PKCS7 core for using PKCS7 formatted signatures.
```

Comment:
Do we want to do this (below) here and elsewhere?
```suggestion
 * Enable PKCS #7 core for using PKCS #7-formatted signatures.
```